src/main/java/Parser.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Used to parse user's commands entered and execute the command
+ */
+public class Parser {
+    private Storage s;
+    private Ui ui;
+    private TaskList tasks;
```

Comment:
perhaps a better variable would be `taskList` as tasks might imply a collection of task